src/Type/ArrayType/ArrayType.php

```diff
@@ -18,38 +18,78 @@ class ArrayType extends \Consistence\ObjectPrototype
 	/**
 	 * Wrapper for PHP in_array, provides safer default parameter
 	 *
+	 * @deprecated
```

Comment:
What about a message? `use ArrayType::hasValue() instead` ? (same for the others)